androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -115,6 +128,11 @@ fun EnhancedVideoPlayer(
                 loop = player.repeatMode == ExoPlayer.REPEAT_MODE_ALL
                 super.onEvents(player, events)
             }
+
+            override fun onTracksChanged(tracks: Tracks) {
+                trackQualityOptions = listOf(autoQualityTrack) + generateTrackQualityOptions(tracks)
```

Comment:
Doesn't `auto` come in the tracks from `generateTrackQualityOptions`?

Reply:
I was doing this initially, do you think it fits better inside `generateTrackQualityOptions`?

Reply:
Well, since we are using a string resource and it will require reading from the context, you can leave this as it is and we'll refine later on